python/ray/rllib/agents/ppo/ppo.py

```diff
@@ -99,6 +99,12 @@ def _init(self):
 
     @override(Agent)
     def _train(self):
+        if "observation_filter" not in self.raw_user_config:
+            # TODO(ekl) remove this message after a few releases
+            logger.info(
+                "Important! Since 0.7.0, observation normalization is no "
+                "longer enabled by default. To enable running-mean "
+                "normalization, set 'observation_filter': 'MeanStdFilter'.")
```

Comment:
Maybe release at most 3 times and then add a note that this message can be ignored if learning performs as expected?

Reply:
ok done